test/e2e/serial/tests/resource_accounting.go

```diff
@@ -288,65 +294,108 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload resourc
-			reqResources[corev1.ResourceCPU] = reqCpu
 
-			reqMem := reqResources[corev1.ResourceMemory]
-			reqMem.Add(excessRes[corev1.ResourceMemory])
-			reqResources[corev1.ResourceMemory] = reqMem
+			By("prepare the test's pod resources as maximum available resources on the target node with the baselaod deducted")
+			baseload.Deduct(reqResources)
 
-			By("Padding all other candidate nodes")
+			By("padding all other candidate nodes leaving room for the baseload only")
 			var paddingPods []*corev1.Pod
 			for _, nodeName := range nrtCandidateNames.List() {
 				node := &corev1.Node{}
 				nodeKey := client.ObjectKey{Name: nodeName}
-				err := fxt.Client.Get(context.TODO(), nodeKey, node)
+				err = fxt.Client.Get(context.TODO(), nodeKey, node)
 				Expect(err).NotTo(HaveOccurred())
 
-				// Padding all the other nodes until only maxResources is free
-				// remember reqResources = maxReources + excessResources
-				// so we ensure out pod will not be scheduled on any other node.
-				padPod, err := makeNodePaddingPod(fxt.Namespace.Name, *node, maxResources)
-				if errors.Is(err, e2enrt.ErrNotEnoughResources) {
-					klog.Infof("Node %q has not enough resources without padding", nodeName)
-					continue
+				//calculate base load on the node
+				baseload, err := nodes.GetLoad(fxt.K8sClient, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", nodeName)
+				By(fmt.Sprintf("computed base load: %s", baseload))
+
+				//get nrt info of the node
+				By(fmt.Sprintf("preparing node %q to fit the test case", nodeName))
+				nrtInfo, err := e2enrt.FindFromList(nrtCandidates, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing NRT info for %q", nodeName)
+
+				baseloadRes := corev1.ResourceList{
+					corev1.ResourceCPU:    baseload.CPU,
+					corev1.ResourceMemory: baseload.Memory,
 				}
-				Expect(err).NotTo(HaveOccurred())
 
-				pinnedPadPod, err := pinPodToNode(padPod, nodeName)
-				Expect(err).NotTo(HaveOccurred())
+				// assuming the node's base load is available under the numas, deduct it from any one and fully pad the rest
+				baseloadZone := nrtInfo.Zones[0]
+				By(fmt.Sprintf("selected zone %q to account for the baseload", baseloadZone.Name))
 
-				err = fxt.Client.Create(context.TODO(), pinnedPadPod)
-				Expect(err).NotTo(HaveOccurred())
+				numaCpu, _ := e2enrt.FindResourceAvailableByName(baseloadZone.Resources, corev1.ResourceCPU.String())
+				numaMem, _ := e2enrt.FindResourceAvailableByName(baseloadZone.Resources, corev1.ResourceMemory.String())
+				availableOnNuma := corev1.ResourceList{
+					corev1.ResourceCPU:    numaCpu,
+					corev1.ResourceMemory: numaMem,
+				}
+
+				for resName, resQty := range availableOnNuma {
+					Expect(resQty.Cmp(baseloadRes[resName])).Should(BeNumerically(">=", 0), "resource %q in base load is %s and is greater than the avialable quantity on zone %q, ", resName, baseloadRes[resName], baseloadZone.Name, resQty)
```

Comment:
this is a good check, but feels misplaced. It should be enforced elsewhere. Let's remove it for now.